components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -200,21 +200,34 @@ func (r *OpenshiftNotebookReconciler) Reconcile(ctx context.Context, req ctrl.Re
 func (r *OpenshiftNotebookReconciler) createProxyConfigMap(notebook *nbv1.Notebook,
 	ctx context.Context) error {
 
-	trustedCAConfigMap := &corev1.ConfigMap{
+	desiredTrustedCAConfigMap := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "trusted-ca",
 			Namespace: notebook.Namespace,
 			Labels:    map[string]string{"config.openshift.io/inject-trusted-cabundle": "true"},
 		},
 	}
 
-	err := r.Client.Create(ctx, trustedCAConfigMap)
+	foundTrustedCAConfigMap := &corev1.ConfigMap{}
+	err := r.Client.Get(ctx, client.ObjectKey{
+		Namespace: desiredTrustedCAConfigMap.Namespace,
+		Name:      desiredTrustedCAConfigMap.Name,
+	}, foundTrustedCAConfigMap)
 	if err != nil {
-		if apierrs.IsAlreadyExists(err) {
-			return nil
+		if apierrs.IsNotFound(err) {
+			r.Log.Info("Creating trusted-ca configmap")
+			err = r.Create(ctx, desiredTrustedCAConfigMap)
```

Comment:
New this code block
should this be :
```suggestion
			err = r.Client.Create(ctx, desiredTrustedCAConfigMap)
```

Reply:
Fixed